src/vos/vos_layout.h

```diff
@@ -273,6 +273,12 @@ struct vos_cont_df {
 	struct vos_gc_bin_df		cd_gc_bins[GC_CONT];
 	/* The epoch for the most new DTX entry that is aggregated. */
 	uint64_t			cd_newest_aggregated;
+	/*
+	 * The epoch for VOS aggregation and incremental reintegration boundary,
+	 * i.e. the data below this epoch are being regarded as valid, and do
+	 * not need to be rebuild during reintegration.
+	 */
+	uint64_t			cd_commit_epoch;
```

Comment:
Don't we need to provide a vos_pool_upgrade plugin to reallocate the container?

Reply:
Indeed, unfortunately, the pool upgrade process does not support the container durable format upgrade yet. sigh, might need add that.

Reply:
Indeed, it will be needed for this to work